rule-003/src/app.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import Koa from 'koa';
```

Comment:
We don't need REST server anymore. ref. Openfaas discussions.

Reply:
We're not deploying on gRPC only now. that's another topic that needs to be discussed.